PyFunceble/cli/__init__.py

```diff
@@ -272,7 +273,7 @@ def tool():  # pragma: no cover pylint: disable=too-many-branches,too-many-state
                     "--database-type",
                     type=str,
                     help="Tell us the type of database to use. "
-                    "You can choose between the following: `json|mariadb|mysql` %s"
+                    "\nYou can choose between the following: `json|mariadb|mysql` %s"
```

Comment:
s/json|mariadb|mysql/json, mariadb, mysql/

_Note:_ suggestion only for enhancing copy/paste facility

Reply:
Could actually also just be

`s/json|mariadb|mysql/json | mariadb | mysql/`This would keep the "regex" explaining, but also leave the room around the option for 
1. Readability
1. Easier copy/paste